ninja-jaxy-routes/src/main/java/ninja/jaxy/JaxyRoutes.java

```diff
@@ -207,7 +207,10 @@ public int compare(Method m1, Method m2) {
         Set<Method> methods = Sets.newLinkedHashSet();
 
         methods.addAll(reflections.getMethodsAnnotatedWith(Path.class));
-        Reflections annotationReflections = new Reflections("", new TypeAnnotationsScanner(), new SubTypesScanner());
+	    Reflections annotationReflections = new Reflections(new ConfigurationBuilder()
```

Comment:
2 tiny things
1) There are tabs - please change that to spaces - we should keep the code base uniform...
2) Can you think of a good way to "test" the functionality? In other words - how can we prohibit someone to replace the new code with the old one without us noticing the mistake?